0 General/01 Course Overview.md

```diff
@@ -250,3 +283,54 @@ At PDX Code Guild we seek to create an environment conducive to personal and pro
 Do not look at the solution or other students' solutions until you've finished the lab yourself. Do not copy other student's code. Do not copy code off the internet unless it's a small amount and you know exactly what it means. Do not try and complete a lab by googling a tutorial. You're only inhibiting your own learning by doing so. Cheating is also a violation of our code of conduct and can result in expulsion.
 
 You are encouraged to use resources (docs, online, books, each other) to research solutions. If you copy code from the internet or a book, you must understand what it means and how it works. If you're not sure, look up the function in the docs, try running it with a variety of inputs, and try tweaking the syntax, throw in print statements to print the values of variables at different times, etc. You also cannot use a single function which performs the entirety of the solution for you. The early labs are designed to practice string and list operations, using a function that does it all defeats the purpose.
+
+## Logistics
+
+### Schedule
+
+The 14 week course is 70 days long, weekdays 9:30 AM to 4:30 PM, with an hour for lunch. The 16 week course is 80 days, weekdays 6 PM to 9:30 PM. We'll take 10 minute breaks every hour-and-a-half.
+
+### Holidays
+
+PDX Code Guild doesn't hold class sessions on the following holidays. Holidays don't affect the length of the course; the end date is pushed back.
+
+* Independence Day
+* Labor Day
+* Thanksgiving Weekend
+* Christmas Weekend
+* New Years Day
+* Memorial Day
+
+### Contact
+
+You are welcome to email me at any time: [CHANGEME](mailto:CHANGEME). We will also have a channel on the [PDX Code Guild Slack](https://portlandcodeguild.slack.com/).
+
+### Zoom
+
+We will be using Zoom for holding class. This is the closest we can get to an in person classroom with lectures, whiteboards, breakouts, etc. You will be given the link for our daily Zoom meetings that make up class.
+
+### Slack
+
+Slack is a chat application. THis is how I will send out quick updates and how you can ask other students and instructors questions. Keep in mind Slack is ephemeral, and messages will disappear after about two weeks.
+
+### Github
+
+That's this! Your main source of information, resources, and assignments for this class.
+
+## Roles & Responsibilities
+
+### Minimum Criteria to Graduate
+
+- all lab solutions- organized, tested, commented
+- a capstone which demonstrates proficiency in each of the five areas we cover
+- not absent more than 10% of the class days (7 for the 14-week class, 8 for the 16-week class)
+
+### Attendance & Attention
```

Comment:
I feel like this information should either not be included or changed.  Some of the text in here suggests way too much leniency.  I did want to preserve what I have locally to discuss these things.

Reply:
Yeah I agree that we should overhaul the course logistics writeup